fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/materialization/AggregatedMaterializedViewRewriter.java

```diff
@@ -725,13 +725,6 @@ private CallOperator getRollupAggregate(RewriteContext rewriteContext,
             Preconditions.checkState(targetColumn instanceof CallOperator);
             return (CallOperator) targetColumn;
         } else {
-            if (targetColumn instanceof CallOperator) {
```

Comment:
Add a TODO, explain three cases we has discussed
1. query metrics matches mv metrics;
2. query metrics constructed from mv dimensions;
3. query metrics constructed from both mv dimensions and metrics hybridly

Reply:
Can I add it into this issue: https://github.com/StarRocks/starrocks/issues/46750